eq-author-api/repositories/strategies/routingStrategy.js

```diff
@@ -368,6 +426,17 @@ const handleAnswerDeleted = (trx, answerId) =>
     }
   );
 
+const handleAnswerCreated = (trx, answer) =>
```

Comment:
Adding a second answer to a question updates the routing rule. I presume that is not right.
We really only want to do this when the condition did not have an answer.

Reply:
It should only refer to the first question. Something isn't working as expected. I'll have a look.